ViewLayer/DebugDisplay/MPPTPage/MPPTView/MpptView.h

```diff
@@ -8,6 +8,7 @@
 #include "../MPPTUi/I_MpptUi.h"
 
 class MpptPresenter;
+class I_MpptUi;
 
 class MpptView : public QObject
```

Comment:
Is `QObject still needed?` you should be able to just have it inherit `I_MpptUi` and it should work, otherwise right now `I_MpptUi` isn't doing anything.